test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Do you need this?

Reply:
no

Reply:
i mean file `user_test` ? is is looks empty